File: app/src/main/java/com/adedom/covid19_th/Covid19Response.java
package com.adedom.covid19_th;

import com.google.gson.annotations.SerializedName;

public class Covid19Response {

    @SerializedName("Confirmed")
    private int confirmed;

    @SerializedName("Recovered")
    private int recovered;

    @SerializedName("Hospitalized")
    private int hospitalized;

    @SerializedName("Deaths")
    private int deaths;

    @SerializedName("NewConfirmed")
    private int newConfirmed;

    @SerializedName("NewRecovered")
    private int newRecovered;

    @SerializedName("NewHospitalized")
    private int newHospitalized;

    @SerializedName("NewDeaths")
    private int newDeaths;

    @SerializedName("UpdateDate")
    private String updateDate;

    @SerializedName("Source")
    private String source;

    public int getConfirmed() {
        return confirmed;
    }

    public int getRecovered() {
        return recovered;
    }

    public int getHospitalized() {
        return hospitalized;
    }

    public int getDeaths() {
        return deaths;
    }

    public int getNewConfirmed() {
        return newConfirmed;
    }

    public int getNewRecovered() {
        return newRecovered;
    }

    public int getNewHospitalized() {
        return newHospitalized;
    }

    public int getNewDeaths() {
        return newDeaths;
    }

    public String getUpdateDate() {
        return updateDate;
    }

    public String getSource() {
        return source;
    }

}

File: app/src/main/java/com/adedom/covid19_th/data/response/Covid19Response.kt
package com.adedom.covid19_th.data.response

import com.google.gson.annotations.SerializedName

data class Covid19Response(
        @SerializedName("Confirmed") val confirmed: Int? = null,
        @SerializedName("Recovered") val recovered: Int? = null,
        @SerializedName("Hospitalized") val hospitalized: Int? = null,
        @SerializedName("Deaths") val deaths: Int? = null,
        @SerializedName("NewConfirmed") val newConfirmed: Int? = null,
        @SerializedName("NewRecovered") val newRecovered: Int? = null,
        @SerializedName("NewHospitalized") val newHospitalized: Int? = null,
        @SerializedName("NewDeaths") val newDeaths: Int? = null,
        @SerializedName("UpdateDate") val updateDate: String? = null,
        @SerializedName("Source") val source: String? = null,
)
